llvm/lib/Object/GOFFObjectFile.cpp

```diff
@@ -24,6 +25,94 @@
 using namespace llvm::object;
 using namespace llvm;
 
+namespace {
+// Return the type of the record.
+GOFF::RecordType getRecordType(const uint8_t *PhysicalRecord) {
```

Comment:
nit: prefer `static` for functions. See https://llvm.org/docs/CodingStandards.html#anonymous-namespaces